soundcheck/src/app/globals.css

```
@@ -0,0 +1,13 @@
+@tailwind base;
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)